client/src/List/List.tsx

```diff
@@ -157,7 +157,7 @@ export default class ListWithItems extends Component<Props, State> {
     api
       .list(
         model,
-        { offset, limit, q: searchTerm },
+        { offset, limit, search: { term: searchTerm, scope: "title" } },
```

Comment:
Is this a breaking change? Or was `q` just used internally in the client?

Reply:
No, this is not a beaking change. `q` has only been used internally.